functionaltests/src/test/java/specs/scheduler/ScheduledJobFixtureHelper.java

```diff
@@ -19,29 +23,30 @@
  */
 public class ScheduledJobFixtureHelper {
 
-    private static Logger LOG = LoggerFactory.getLogger(ScheduledJobShortFixture.class);
-
     private final BasePaymentFixture paymentFixture;
+    private final Logger logger;
 
     public ScheduledJobFixtureHelper(BasePaymentFixture paymentFixture) {
         this.paymentFixture = paymentFixture;
+        this.logger = LoggerFactory.getLogger(paymentFixture.getClass());
```

Comment:
why change the logger's name to the class of the `paymentFixture` instance class instead of this class' (`ScheduledJobFixtureHelper`) name? wouldn't it be confusing while debugging/checking the logs?

Reply:
exactly to avoid confusion: `ScheduledJobFixtureHelper` is re-used in several classes, but i want to know which actual class (`ScheduledJobShortFixture` or `ScheduledJobShortFixture2`) is logging right now.

Reply:
ok